packages/sync/composer.json

```diff
@@ -10,6 +10,9 @@
 		"automattic/jetpack-roles": "@dev",
 		"automattic/jetpack-status": "@dev"
 	},
+	"require-dev": {
+		"automattic/jetpack-scripts": "@dev"
```

Comment:
Should every package really have the same dev dependency, or can we have that dependency just for the master package?

Reply:
To update the dependency tree for the root package, it needs to be included only in root dev dependencies. But since we, theoretically, will update all our packages independently it might be good idea to add it to all of our packages.

Reply:
OK, so if your plan is to make it work for both the monorepo and individual packages, I'm happy :)